rpc/swap.go

```diff
@@ -121,7 +121,7 @@ func (s *SwapService) GetPast(_ *http.Request, req *GetPastRequest, resp *GetPas
 	}
 
 	sort.Slice(resp.Swaps, func(i, j int) bool {
-		return resp.Swaps[i].StartTime.UnixNano() < resp.Swaps[j].StartTime.UnixNano()
+		return resp.Swaps[j].StartTime.UnixNano() < resp.Swaps[i].StartTime.UnixNano()
```

Comment:
I know you didn't write the original code, but `Time` has built-in methods `Before` and `After` that are the preferred way to do comparisons. I don't think the extra efficiency matters much here, but switching to the preferred methods might keep this style from spreading further in to the code base.